cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts

```diff
@@ -675,68 +731,12 @@ describe('Asset Properties Widget: Configuration/View screen tests', function ()
     cy.get(assetNameElement).eq(4).should('contain.text', 60);
     cy.get(assetNameElement).eq(5).should('contain.text', 10);
     cy.get(assetNameElement).eq(6).should('contain.text', 15);
+    cy.wait(1000); // Added wait to resolve the flakyness
```

Comment:
Verify if adding an intercept solves the issue.

Reply:
Added intercept